### DISCUSSION TITLE
In 1.4, the taxable sales reports (AR->Reports->(non)Taxable sales) have disappeared

### DISCUSSION THREAD

**Discussion by ehuelsmann** [ledgersmb/LedgerSMB#5819] [Design]
Reported on IRC, the reports which used to be in AR->Reports->(non)Taxable sales in 1.3, are nowhere to be found in the 1.4 menu.

**Comment by ehuelsmann** [comment]
Talking with Chris, as it turns out we decided that the answers from the reports were incorrect to begin with, so we removed them. However, there now has to be a (documented) alternative.

Hopefully we get it documented soon, or have an alternative implemented by 1.5.

**Comment by einhverfr** [comment]
I believe even SQL-Ledger removed some of these reports, btw.
I would like a better report that can arise from the financia rewrite (so leaving it open in that regard)

**Comment by einhverfr** [comment]
Moving to 1.6 for now due to need for more financial design

**Comment by einhverfr** [comment]
Tagging needs design and experimental due to questions of global requirements.

**Comment by sbts** [comment]
This has been raised recently in Australia

**Comment by ehuelsmann** [comment]
@sbts could you list the requirements somewhere for reference?

**Comment by sbts** [comment]
@ehuelsmann as you have asked in #ledgersmb @cornercase is going to be best able to outline the requirements

**Comment by ehuelsmann** [comment]
@sbts , @cornercase, ping! Could you provide your requirements so I can consider including in 1.9?

**Comment by cornercase** [comment]
@sbts @ehuelsmann I think you might have the wrong user, I don't think I've ever used LedgerSMB, unless you've got an issue report or something to jog my memroy.

**Comment by ehuelsmann** [comment]
@cornercase sorry about that. Indeed we have a different user in mind.

**Comment by ehuelsmann** [comment]
@tonka128, could you provide your requirements so I can consider inclusion in 1.9?

**Comment by tonka128** [comment]
OK - so in Aus everything had GST except a select list of items (dumbest move to make - would be so much cleaner if everything was consistent) - when reporting your tax you are asked how much  of it is without GST.
But then there are differet tyypes of "no GST" - ie someone not registered / or an item that doesnt attract it / or an export sale.
Because of the different types this is normally handled in most packages by creating tax codes with 0% tax and then running reports against those codes.
So - not sure this exact bug is anything thats really a fix.
To make LSMB an absolute godsend in Aus my wish-list would look like:
1. Make sure any AR or AP transaction that has a 0% tax code allocated recorded the Taxable amount against the 0% tax code
2. a simple report that then allowed you to pull up the taxable amounts against any code for a period.
3. *the big one* - A report that shows the COGS associated with a particlar tax codes for a set period (@sbts has this code as a script I run right now)

**Comment by ehuelsmann** [comment]
Reading the above, I am thinking how to implement this in LedgerSMB. @tonka128, a few questions come up for (2):

1. Does it only concern the amounts per invoice, or is it relevant which *lines* are taxed on that invoice? I'm asking because manual taxes don't keep a correspondence relationship between the "tax base amount" and the lines.
2. Does your company report on cash or on accrual basis? I'm asking, because if it's on cash basis, we'd need a rule to determine the tax amount on the invoice applicable to any partially paid invoices

With respect to (1), I'm wondering whether there are multiple 0% rates and whether a customer/part combination can have multiple 0% rates applicable. And if so, do we accept that multiple match? Or do we need "conflict resolution": does one need to be picked over the others?

**Comment by tonka128** [comment]
1. Needs to be calculated on a line by line as per convergence points - but for the tax type reports - really you just need the totals amount that was allocated agains that tax code (not the calcualted $0)
2. Aus allows both cash and accrual - we choose accrual for the types of reasons you have pointed out - I would find it acceptable to say this all works great in Aus - must choose accrual.
Yes there are multiple 0% rates - as long as it has been set up correctly there should only be a single tax convergance point for any single line item.
